src/schema/v2/__tests__/gene.test.js

```diff
@@ -2,7 +2,8 @@
 import { runQuery } from "schema/v2/test/utils"
 import trackedEntityLoaderFactory from "lib/loaders/loaders_with_authentication/tracked_entity"
 
-describe("Gene", () => {
+// FIXME: Yeah, this is whack
```

Comment:
😺

Reply:
cc @alloy: This is one of those tests where I didn't know how to effectively port it.

Reply:
Cool, will take a stab at this in a follow-up 👍